website/docs/docs/quickstarts/dbt-cloud/redshift-qs.md

```diff
@@ -0,0 +1,285 @@
+    </div>
+
+6. Click **Test Connection**. This verifies that dbt Cloud can access your Redshift cluster.
+7. Click **Next** if the test succeeded. If it failed, you might need to check your Redshift settings and credentials.
+
+## Set up a dbt Cloud managed repository 
+<Snippet src="tutorial-managed-repo" />
+
+## Initialize your dbt project​ and start developing
+Now that you have a repository configured, you can initialize your project and start development in dbt Cloud:
+
+1. Click **Develop** from the upper left. It might take a few minutes for your project to spin up for the first time as it establishes your git connection, clones your repo, and tests the connection to the warehouse.
+2. Above the file tree to the left, click **Initialize dbt project**. This builds out your folder structure with example models.
+3. Make your initial commit by clicking **Commit & Sync**. Use the commit message `initial commit` and click **Commit**. This creates the first commit to your managed repo and allows you to open a branch where you can add new dbt code.
+4. You can now directly query data from your warehouse and execute `dbt run`. You can try this out now:
+    - In the IDE's editor, add this query: 
+        ```sql
+        select * from jaffle_shop.customers
+        ```
+    - In the command line bar at the bottom, enter `dbt run` and click **Enter**. You should see a `dbt run succeeded` message.
+
+## Build your first model
+1. Click **Develop** from the upper left of dbt Cloud. You need to create a new branch since the main branch is now set to read-only mode. 
+2. Click **Create branch**. You can name it `add-customers-model`.
+3. Click the **...** next to the Models directory, then select **Create file**.  
+4. Name the file `models/customers.sql`, then click **Create**.
+5. Copy the following query into the file and click **Save**.
+
+```sql
+with customers as (
+
+    select
+        id as customer_id,
+        first_name,
+        last_name
+
+    from jaffle_shop.customers
+
+),
+
+orders as (
+
+    select
+        id as order_id,
+        user_id as customer_id,
+        order_date,
+        status
+
+    from jaffle_shop.orders
+
+),
+
+customer_orders as (
+
+    select
+        customer_id,
+
+        min(order_date) as first_order_date,
+        max(order_date) as most_recent_order_date,
+        count(order_id) as number_of_orders
+
+    from orders
+
+    group by 1
+
+),
+
+final as (
+
+    select
+        customers.customer_id,
+        customers.first_name,
+        customers.last_name,
+        customer_orders.first_order_date,
+        customer_orders.most_recent_order_date,
+        coalesce(customer_orders.number_of_orders, 0) as number_of_orders
+
+    from customers
+
+    left join customer_orders using (customer_id)
+
+)
+
+select * from final
+```
+
+6. Enter `dbt run` in the command prompt at the bottom of the screen. You should get a successful run and see the three models.
```

Comment:
This fails for me but I think it's related to the `select * from jaffle_shop.customers` step above and the uncertainty of where to put that info based on the vague instructions.